### PR TITLE
added recipe for simpleperceptron.

### PR DESCRIPTION
**Pull request by wiphoo**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by wiphoo** [comment]
@conda-forge/help-python this recipe is ready for review.

File: recipes/simpleperceptron/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - simpleperceptron
+
+about:
+  home: https://github.com/wiphoo/neural-simple_perceptron
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Simple for experimental on neual network base on perceptron for Python'
```
Comment (wolfv) [inline recipes/simpleperceptron/meta.yaml]: typo

File: recipes/simpleperceptron/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  description: |
+    simple_perceptron is a simple, easy, complete, for experimental on neural network.
+    this package including Perceptron object to contruct the network, 
+    Forward/Backward propagation to do an calculation the error 
+    and ActivationFuncs to apply nnon-linear in network.
+  doc_url: https://github.com/wiphoo/neural-simple_perceptron/examples
```
Comment (wolfv) [inline recipes/simpleperceptron/meta.yaml]: URL doesn't work

**Comment by wolfv** [comment]
To be honest, I looked at the source code, and I am not sure if this package is mature enough to be included into the `conda-forge` distribution. 

How about publishing it into your own channel? Just build the package and use the upload functionality :)

**Comment by wiphoo** [comment]
> To be honest, I looked at the source code, and I am not sure if this package is mature enough to be included into the `conda-forge` distribution.
> 
> How about publishing it into your own channel? Just build the package and use the upload functionality :)

understand and thank you for feedback. I will improve and hopefully, it will mature enough to included in `conda-forge` channel

**Pull request closed without merging**